extruct/jsonld.py

```diff
@@ -16,26 +16,24 @@
 class JsonLdExtractor(object):
     _xp_jsonld = lxml.etree.XPath('descendant-or-self::script[@type="application/ld+json"]')
 
-    def extract(self, htmlstring, base_url=None, encoding="UTF-8"):
+    def extract(self, htmlstring, base_url=None, encoding="UTF-8", as_json=False):
```

Comment:
To me ``as_json`` sounds confusing - it's not clear what will this return, parsed JSON or not parsed (I'd lean towards parsed, although in reality it's not parsed). I'm not sure which name would be better, maybe ``parse_json=True``?

Reply:
I agree parse_json sounds nicer!